core/src/main/scala/io/finch/package.scala

```diff
@@ -1,12 +1,14 @@
 package io
 
+import cats.effect.IO
+
 /**
  * This is a root package of the Finch library, which provides an immutable layer of functions and
  * types atop of Finagle for writing lightweight HTTP services.
  */
-package object finch extends Endpoints
-    with Outputs
-    with ValidationRules {
+package object finch {
+
+  object catsEffect extends Module[IO]
```

Comment:
Maybe we don't have to provide this?

Reply:
I think having some out-of-the-box example should be useful. `IO` comes for free as a data type inside of `cats-effect` anyway